RAMP/ramp.py

```diff
@@ -98,6 +98,7 @@ def unpack(bundle):
 @click.option('--runcmdargs', 'run_command_line_args', default=None, help='if set, use those args when running the redis server')
 @click.option('--redis-min-version', '-r', 'min_redis_version', default=None, help='redis minimum version')
 @click.option('--redis-pack-min-version', '-R', 'min_redis_pack_version', default=None, help='redis pack minimum version')
+@click.option('--compatible_redis_version', '-cr', 'compatible_redis_version', default=None, help='redis compatible version')
```

Comment:
Make it similar to `--redis-min-version` ?
```suggestion
@click.option('--redis-compatible-version', '-cr', 'compatible_redis_version', default=None, help='redis compatible version')
```